examples/cc3200/xively_demo/.project

```diff
@@ -59,7 +59,7 @@
 	<variableList>
 		<variable>
 			<name>CC3200_SDK_ROOT</name>
-			<value>file:/C:/ti/tirex-content/CC3200SDK_1.1.0/cc3200-sdk</value>
+			<value>file:/C:/ti/CC3200SDK_1.1.0/cc3200-sdk</value>

```

Comment:
new default path for these installations have removed hte tirex-content directory.